include/private/vkd3d_common.h

```diff
@@ -81,6 +82,36 @@ static inline bool vkd3d_bitmask_is_contiguous(unsigned int mask)
     return vkd3d_popcount(mask) == j;
 }
 
+/* Returns 64 for mask == 0 */
+static inline unsigned int vkd3d_bitmask_tzcnt64(uint64_t mask)
+{
+#ifdef _MSC_VER
+    unsigned long result;
+    _BitScanForward64(&result, mask) ? result : 64;
+    return result;
+#elif defined (HAVE_BUILTIN_CTZLL)
```

Comment:
Is this activated in autoconf?

Reply:
Apparently I forgot to commit the change.